tags/tests/test_link_tag.py

```diff
@@ -0,0 +1,96 @@
+import json
+
+from tags.tests.base_test import BaseTestCase
+
+
+class TagNamespaceTestCase(BaseTestCase):
+    fixtures = ["test_authentication.yaml", "test_marketplace.yaml", "test_tags.yaml"]
+
+    def test_add_global_tag(self):
+        self.login("17simple")
+        res = self.post("/tags/link/product/2/tag/1/")
+        self.assertStatusCode(res, 201)
+
+        res = self.get("/associations/products/2/")
+        self.assertStatusCode(res, 200)
+        self.assertEqual(
+            json.loads(res.content).get("tags"),
```

Comment:
I think it applies to your case, usually `res.data` contains the deserialized Python objects, so you don't have to use `json.loads`